api/provider/aws/load_balancer_read.go

```diff
@@ -33,7 +33,7 @@ func (l *LoadBalancerProvider) Read(loadBalancerID string) (*models.LoadBalancer
 			// certificate arn format:  arn:aws:iam:region:012345678910:certificate/path/to/name
 			split := strings.SplitN(certificateARN, "/", -1)
 			certificateName := split[len(split)-1]
-			port.CertificateName = certificateName
+			port.Certificate = certificateName
```

Comment:
This can cause diffs in terraform to fail, since a user may specify a certificate name and get back a certificate arn. Couple options:
1. Update the terraform code to suppress (or fix) the diff
2. Only allow ARNs wherever we have certificates

Reply:
Personally, I think requiring users to use ARNs for certificates would follow the convention of "be more explicit" and seems to be a better option, though that would require users to update terraform files to change out cert names with ARNs.